bundles/org.openhab.binding.sunsynk/src/main/java/org/openhab/binding/sunsynk/internal/config/SunSynkInverterConfig.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.openhab.binding.sunsynk.internal.config;
+
+/**
+ * The {@link SunSynkInverterConfig} Parameters used for Inverterconfiguration.
+ *
+ * @author Lee Charlton - Initial contribution
+ */
+
```

Comment:
Annotate nonnulbydefault

Reply:
Not sure I can do this, any help  apricated.
In SunSynkAccountHandler:  
`Unsafe interpretation of method return type as '@NonNull' based on substitution 'T=@NonNull Details'. Declaring type 'Gson' doesn't seem to be designed with null type annotations in mind`
gson.fromJson is only attempted in a try catch.